src/ucp/wireup/wireup.c

```diff
@@ -620,6 +623,73 @@ ucp_wireup_process_reply(ucp_worker_h worker, ucp_ep_h ep,
     }
 }
 
+static void ucp_wireup_close_flushed_callback(ucp_request_t *req)
```

Comment:
duplicate of the code in ucp_ep.c?

Reply:
partially: only scheduling of disconnect progress.
move to separate function?

Reply:
yes, extract common code to helper, and use different function names